config/install/acquia_lift.settings.yml

```diff
@@ -17,4 +17,5 @@ thumbnail: []
 visibility:
   # Track, only when the page URL falls OUTSIDE OF these path patterns.
   path_patterns: "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*/*\n/block/*"
+content_replacement_mode: 'untrusted'
```

Comment:
Currently, there is a consistency between the actual settings object and their menu representation. To keep this consistency, there should be an "advanced" parent setting, and then its "content_replacement_mode" child setting. This means I am recommending:


`advanced:`
`  content_replacement_mode: 'untrusted'`

P.S. I am concerned about the "Advanced" wording, but will note in a separate comment.